core/domain/topic_services_test.py

```diff
@@ -698,6 +698,43 @@ def test_manager_cannot_assign_roles(self):
         self.assertFalse(topic_services.check_can_edit_topic(
             self.user_b, topic_rights))
 
+    def test_get_all_topic_rights_of_user(self):
+        topic_services.assign_role(
+            self.user_admin, self.user_a,
+            topic_domain.ROLE_MANAGER, self.TOPIC_ID)
+        topic_rights = topic_services.get_topic_rights_with_user(self.user_id_a)
+        self.assertEqual(len(topic_rights), 1)
+        self.assertEqual(topic_rights[0].id, self.TOPIC_ID)
+        self.assertEqual(topic_rights[0].manager_ids, [self.user_id_a])
+
+    def test_deassign_user_from_all_topics(self):
```

Comment:
Add test doing this where a user has manager access to 2+ topics (to make sure the query / for-loop works correctly)

Reply:
Done